src/sentry/interfaces/stacktrace.py

```diff
@@ -434,7 +434,7 @@ def to_json(self):
             'colno': self.colno
         })
 
-    def get_hash(self, platform=None):
+    def compute_hashes(self, platform=None):
```

Comment:
note the change from `get_hash` to `compute_hashes` here which is why the return value changes to another layer of list